app/views/layouts/application.html.erb

```diff
@@ -48,9 +48,40 @@
         <% end %>
       </div>
 
-    <% if current_user %>
-      <% if FeatureService.enabled?(:new_publish_navigation) && current_user.has_multiple_providers_in_current_recruitment_cycle? && !@provider.nil? && !request.path.include?('support') %>
-       <%= render TitleBar::View.new(title: @provider.provider_name) %>
+    <% if FeatureService.enabled?(:new_publish_navigation) %>
+
+      <% if current_user %>
+        <% if current_user.has_multiple_providers_in_current_recruitment_cycle? && @provider && !request.path.include?('support') %>
+         <%= render TitleBar::View.new(title: @provider.provider_name) %>
+        <% end %>
+      <% end %>
+
+      <% if @provider && !current_page?(root_path) && !@provider.rolled_over? && @provider.accredited_body? && @provider.recruitment_cycle && !request.path.include?('support') %>
```

Comment:
There's a bit of a logic soup here and a couple ways to clean this up:

1. Put the different nav views in partials (conditional methods should probably go into helpers)
2. Put them into sep view components and encapsulate their conditional logic in their `render?` methods